src/options/parser.rs

```diff
@@ -91,7 +91,7 @@ pub enum Strictness {
 
 /// Whether a flag takes a value. This is applicable to both long and short
 /// arguments.
-#[derive(Copy, Clone, PartialEq, Debug)]
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
```

Comment:
Maybe change this to:

```suggestion
#[derive(PartialEq, Eq, Debug, Copy, Clone)]
```

so that everything use the same order for derive.